### PR TITLE
Milestone 5: mobile menu

### PR DESCRIPTION
**Pull request by Khanetmoi**
Portfolio: mobile menu

In this project I :

- implemented the mobile menu using my recently acquired Javascript knowledge
- added Javascript + HTML + CSS linters
- when the user clicks on the hamburger button, the mobile menu appears
- when the user clicks on any of the mobile options, the mobile menu disappears and the correct part of the website is displayed
- when the user clicks the close (X) button, the mobile menu disappears
- stuck to the Figma design as much as I could

### CODE REVIEW

File: javascript/mobile.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const hambugerButton = document.getElementById('hamburgerButton');
+const hambugerButton2 = document.getElementById('hamburgerButton2');
+const options = document.getElementsByClassName('options');
+
+hambugerButton.addEventListener('click', () => {
+  contactList.classList.toggle('appear');
+  // if(screen.Width > 776)
+  // {
+  //     contactList.style.backgroundImage = "none";
+  // }
```
Comment (youmari) [inline javascript/mobile.js:11]: - [ ] please try to remove the unused code in order to keep the file clean and professional

File: index.html [hunk 2]
```diff
@@ -12,11 +12,34 @@
...
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#About" class="options"> <h3 class="mobile-menu">About</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#contact" class="options"> <h3 class="mobile-menu">Contact</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+            </ul>
```
Comment (youmari) [inline index.html:42]: - [ ] the navigation links are working fine in the mobile version , however they aren't working in the desktop version so please try to fix that

File: index.html [hunk 3]
```diff
@@ -12,11 +12,34 @@
...
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#About" class="options"> <h3 class="mobile-menu">About</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#contact" class="options"> <h3 class="mobile-menu">Contact</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
```
Comment (youmari) [inline index.html:41]: - [ ] the arrows next to links doesn't have the same colors as the ones in the Figma design so , please try to export the arrow in the Figma and use them in your menu in order to stick to the design 

- [ ] Figma design 
image

- [ ] Yours 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Khanetmoi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error
✔️ Great title and short summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gtekle — APPROVED** [review]
Hi @Khanetmoi :wave:, 

# STATUS: APPROVED 🎉🎉🎉 :clap: :clap: :clap: :100: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Got Talent-congrats

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**